docs/release-notes/7.16.asciidoc

```diff
@@ -0,0 +1,54 @@
+[discrete]
+[[release-notes-7.16.0]]
+== 7.16.0
+
+[discrete]
+[[features-7.16.0]]
+==== Features
+* Adds the ability to configure trusted applications on a per-policy basis, allowing security administrators to target a set of hosts with specific configurations and settings. For example, trusted applications could be tailored to certain functions within an organization or for testing and troubleshooting purposes ({pull}112182[#112182], {pull}111051[#111051], {pull}110966[#110966]).
+* Adds memory threat protection for macOS and Linux systems.
+* Provides certified applications for {sn} Security Operations (SecOps) and {sn} IT Service Management (ITSM), and introduces a new {sn} IT Operations Management (ITOM) connector ({pull}105440[#105440], {pull}114125[#114125]).
+* Adds logic to finding pending isolation or release statuses for endpoints added in {es-sec} version 7.16 or later ({pull}115441[#115441]).
+* Adds {Fleet} actions and responses to the endpoint activity log and enriches the log by showing successful or failed action responses that were completed when the endpoint finished executing the action request ({pull}114905[#114905]).
+* Resolves deep link IDs ({pull}111984[#111984]).
+* Allows users to create host isolation exceptions ({pull}111253[#111253]).
+* Allows cases to be imported and exported as saved objects ({pull}110148[#110148]).
+* Adds the Risky Hosts view to the Overview page ({pull}109553[#109553]).
+* Adds host risk metadata to alert details ({pull}113274[#113274]).
+
+[discrete]
+[[bug-fixes-7.16.0]]
+==== Bug fixes and enhancements
+* The *Analyze event* option has been moved from the overflow menu to the Actions column. It now only displays events that can be opened in the visual event analyzer ({pull}115478[#115478]).
+* Halts indicator match rule execution after the allotted time interval has passed ({pull}115288[#115288]).
+* Allows migration of detection rule actions to centralized Kibana alerting framework. Users may receive notifications sooner after alerts have been generated, depending on rule configuration and actions frequency ({pull}115243[#115243], {pull}115101[#115101]).
+* Changes the prebuilt indicator match rule's interval and lookback time to one hour ({pull}115185[#115185]).
+* Allows exceptions to be exported with rules ({pull}115144[#115144]).
+* Improves the formatting of array values and JSON in the *Table* and *JSON* tabs ({pull}115141[#115141]).
+* Provides users with a new, simpler way to add data to their environments through the {agent} ({pull}115016[#115016], {pull}112142[#112142]).
```

Comment:
```suggestion
* Redesigns the welcome pages that appear when no data is present ({pull}115016[#115016], {pull}112142[#112142]).
```
Maybe I am misreading the linked issues, but I would describe this as something like "Redesigns the welcome pages that appear when no data is present".

Reply:
I'm leaning towards leaving this as is because the summary came from @kevinlog. Though, I'm completely open to revising this @kevinlog if you want to blend the two.